### PR TITLE
Documentation for Slovenian eID (EOI)

### PR DESCRIPTION
**Pull request by msetina**
Added documentation for Slovenian eID and some changes <pre><code> to .bc textile markup.

### CODE REVIEW

File: Slovenian-EID.textile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Slovenian EID (eOI) is a card manufactured by NXP. They also provide the software solution IDprotect adopted for SI-TRUST. There are versions for Windows and Mac accessible from "https://www.si-trust.gov.si/sl/eoi/programska-oprema-idprotect-client/":https://www.si-trust.gov.si/sl/eoi/programska-oprema-idprotect-client/
+
+Support for Linux was planned at the begining, but with time it was removed.
+
+h2. The card
+
+The eOI has contact and contactless support over NFC. NFC requires CAN to enable access. On the card there are multiple apps for different levels of security. First app (E828BD080F014E585031) has one token for low security use that does not require PIN to access.
+Second app (E828BD080F014E585030) has two high security tokens. One token for identification and one token for signing. Both need PIN to access. 
+All tokens on the card are 384 bit EC. All tokens have SI-TRUST as issuer. CA and ROOT certs are on the card. ROOT cert is self signed.
```
Comment (Jakuje) [inline Slovenian-EID.textile:11]:
```suggestion
All keys on the card are 384 bit EC. All certificates have SI-TRUST as issuer. CA and ROOT certs are on the card. ROOT cert is self signed.
```

File: Slovenian-EID.textile [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+With no configuration all apps and tokens are available to the user with PKSC11 interface. To use authentication cert with a browser, you need configure PKSC15 framework to enable just the app (E828BD080F014E585030) containing authentication token (Podpis in prijava (Norm PIN)).
+
+With OpenSC apps can be configured separately. To access high security tokens in browser the opensc.conf should include the folowing in app default section:
+
+bc.. framework pkcs15 {
+                # Slovenian eID - low level (pinless, "Prijava brez PIN-a")
+                application E828BD080F014E585031 {
+                model = "ChipDocLite";
+                disable = true;
+                user_pin = "Norm PIN";
```
Comment (Jakuje) [inline Slovenian-EID.textile:26]: can you indent this block to increase the readability? The same for the following blocks

**Comment by msetina** [comment]
I made the suggested and found some more formating problems and fixed them. I do not have a Fedora based system, so I just rewritten to sudo and dnf.

**Review by Jakuje — APPROVED** [review]
Thank you! Looks good now. I will let others to have a look too.

Just a note that the wiki supports also markdown if you are more familiar with it. I like its formatting slightly more intuitive and preview works in the pull requests.

**Review by xhanulik — APPROVED** [review]

**Pull request merged**